docs/general/installation/truenas.md

```diff
@@ -0,0 +1,534 @@
+This will update the custom app's icon with Jellyfin's logo.
+
+---
+
+## Installing with the SCALE Community App
+
+To install the **Jellyfin** application, go to **Apps**, click **Discover Apps**, either begin typing Jellyfin into the search field or scroll down to locate the **Jellyfin** application widget.
+You might need to click "Refresh Catalog" if no Jellyfin app is showing in the results.
+
+![Jellyfin App Widget](/images/docs/install-truenas-8.png)
+
+Click on the widget to open the **Jellyfin** application details screen.
+
+![Jellyfin App Details Screen](/images/docs/install-truenas-9.png)
+
+Click **Install** to open the Jellyfin application configuration screen.
+
+Application configuration settings are presented in several sections, each explained below.
+To find specific fields, click in the **Search Input Fields** search field, scroll down to a particular section or click on the section heading on the navigation area in the upper-right corner.
+
+![Install Jellyfin Screen](/images/docs/install-truenas-10.png)
+
+### Application Name Settings
+
+Accept the default value or enter a name in the **Application Name** field.
+In most cases, use the default name, but if adding a second deployment of the application you must change this name.
+
+Accept the default version number in **Version**.
+When a new version becomes available, the application has an update badge.
+The **Installed Applications** screen shows the option to update applications.
+
+### Jellyfin Configuration Settings
+
+You can accept the defaults in the **Jellyfin Configuration** settings, or enter the settings you want to use.
+
+You can enter a **Published Server URL** for use in UDP autodiscovery, or leave it blank.
+
+If needed, click **Add** to define **Additional Environment Variables**, see [Configuration](https://jellyfin.org/docs/general/administration/configuration/) for options.
+
+### User and Group Settings
+
+You can accept the default value of 568 (apps) in **User ID** and **Group ID** or define your own.
+
+![User and Group Settings](/images/docs/install-truenas-11.png)
+
+This user and group is used for running the Jellyfin container only and cannot be used to log in to the Jellyfin web interface.
+Create an admin user in the Jellyfin initial setup wizard to access the UI.
+
+### Networking Settings
+
+Select **Host Network** under **Network Configuration** if using [DLNA](/docs/general/networking/dlna/), to bind network configuration to the host network settings.
+Otherwise, leave **Host Network** unselected.
+
+![Networking](/images/docs/install-truenas-12.png)
+
+You can accept the default port number in **WebUI Port**, which is **30013**.
+
+You can change this to port **8096**.
+Most Jellyfin clients have built-in scanning features that look for port **8096** by default.
+
+Refer to the TrueNAS [default port list](https://www.truenas.com/docs/references/defaultports/) for a list of assigned port numbers.
```

Comment:
the auto discovery ports should also be configured if possible https://jellyfin.org/docs/general/networking/#static-ports